README.md

```diff
@@ -262,7 +262,7 @@ Make sure that you have the following required dependencies installed:
 
 * Clang-format 6.x or higher
 
-* [PlantUML](http://plantuml.com/) including Graphviz (for building documentation only) 
+* A recent version of [PlantUML](http://plantuml.com/), including Graphviz, for building documentation. See [Documentation](#Documentation) for our recommendations on installing. The version available in common package repositories may be out of date.
```

Comment:
```suggestion
* A recent version of [PlantUML](http://plantuml.com/), including Graphviz, for building documentation. Note that the ubuntu package is likely to old to work properly. See [Documentation](#Documentation) for how to configure it for FPC. 
```
Thought we could be a bit more forceful on need to install from plantuml.com and as docu section currently talks only about the configuration aspect of the installation i correspondingly rephrased a bit

Reply:
Yes, I want to be forceful for this manual type install from plantuml.com

Not sure what specific version would be required, but it is definitely more than the version from 2015 present in ubuntu.